test/front_desk_test.rb

```diff
@@ -0,0 +1,152 @@
+require_relative "test_helper"
+
+describe "FrontDesk" do
+  describe "Find existing reservation" do
+    before do
+      @front_desk = Hotel::FrontDesk.new
+
+      @reservation_one = Hotel::IndividualReservation.new(
+        date_range = Hotel::DateRange.new("2020-03-15", "2020-03-17"), 
+        room_num = 1
+      )
+      @reservation_two = Hotel::IndividualReservation.new(
+        date_range = Hotel::DateRange.new("2020-04-01", "2020-04-05"), 
+        room_num = 2
+      )
+    end
+
+    it "Find the reservation using a date (return the associate reservations)" do
+      @front_desk.add_reservation(@reservation_one)
+      @front_desk.add_reservation(@reservation_two)
+      date = "2020-04-01"
+      expect(@front_desk.check_reservations_on_date(date)).must_be_kind_of Array
+      expect(@front_desk.check_reservations_on_date(date)).must_include @reservation_two
+    end
+
+    it "Find the reservation using a date (none reservation on that day)" do
+      @front_desk.add_reservation( @reservation_one)
```

Comment:
You've got an extra space in some of these `add_reservation` calls.

```suggestion
      @front_desk.add_reservation(@reservation_one)
```